resources/css/site.css

```diff
@@ -603,3 +603,49 @@ a.text-center.btn.btn-outline-light.btn__book {
 .card-img-top {
     height: 100% !important;
 }
+
+.p-heading {
+    font-weight: 500;
+}
+
+.p-container {
+    flex-direction: column-reverse;
+    padding: 40px;
+}
+
+.p-container--single {
+    color:#299942;
+}
+
+.p-container--joint {
+    color: #165181;
+    margin-bottom: 16px;
+}
+
+.p-image {
+    max-width: 30%;
+    margin: 0 auto;
+}
+
+.p-image--single {
+    filter: invert(47%) sepia(44%) saturate(738%) hue-rotate(82deg) brightness(93%) contrast(89%);
+}
+
+.p-image--joint {
+    filter: invert(25%) sepia(77%) saturate(564%) hue-rotate(165deg) brightness(95%) contrast(97%);
+}
+
+.btn-credit,
+.btn-direct {
+    padding: 18px 36px;
+    font-size: 1.15rem;
+}
+
+@media screen and (min-width: 993px) {
+    .btn-credit,
+    .btn-direct {
+        max-width: 85%;
+        width: 100%;
+        margin: 0 auto;
```

Comment:
Hi @ostafinskim , while we don't have CSS style guide for this project yet (or in general), I'd recommend sorting your CSS properties alphabetically, e.g.

```
    .btn-credit,
    .btn-direct {
        margin: 0 auto;
        max-width: 85%;
        width: 100%;
    }
```

I know it might seem really minor, but if you do this consistently, then it makes it easier to skim through large blocks of CSS and find what you want quickly.

There are other ways to group CSS properties (e.g. by type, as shown here: https://medium.com/swlh/better-ways-to-organise-css-properties-9a066e7ded62), but the problem is they are quite complicated to learn and remember. Sorting the properties alphabetically is very easy to remember, and to read through :)

Reply:
Thanks!